### PR TITLE
Add minio bucket folder option

### LINKED ISSUE #27: Support for sub-buckets/subfolder
It would be good to allow sub-bucket as a variable.

### PR DESCRIPTION
**Pull request by JanCaha**
Add option to specify folder within MinIO bucket to store files to. 

Fixes #27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ docker run -it \
   -e MINIO__SECRET_KEY=secret-key \
   -e MINIO__BUCKET=destination-bucket \
   -e MINIO__SECURE=1 \
+  -e MINIO__BUCKET_SUBPATH=SubFolder \
```
Comment (PeterPetrik) [inline README.md:64]: Maybe you can specify in README this is optional?

**Comment by PeterPetrik** [comment]
LGTM

**Comment by JanCaha** [comment]
Added. @PeterPetrik could you handle the Docker release? I dont think I can make releases there.

**Comment by PeterPetrik** [comment]
Best to ask @wonder-sk I do not know how is this released

**Pull request merged**